CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - OTA: New method - `EspFirmwareInfoLoad::fetch_native` - returning the full native ESP-IDF image descriptor structures
 - Added `use_serde` feature, which enables the `use_serde` feature of `embedded-svc` crate, allowing to deserialize configuration structs.
+- Added argument `subprotocol_list` to `ws_handler` to allow subprotocols to be supported by WebSockets
```

Comment:
This is a breaking change, so this should be documented under the "Breaking" section of the changelog.